block/manager.go

```diff
@@ -328,3 +330,22 @@ func (m *Manager) setDA(daconfig string, dalcKV store.KV, logger log.Logger) err
 	m.Retriever = retriever
 	return nil
 }
+
```

Comment:
another edge case I see here is the following: I was a sequencer previously, and than I wasn't (rotation. but continued being a full node), and than I am again. In this case, my store `LastGossipedHeight` wasn't updated when I was a full node (or at least I don't see it). and than I need to get stuck on gossiping potentially a big amount. 

I think we need some sort of a sane skew after which we avoid gossiping (i.e if I'm 1M blocks away from the latest state, than probably it doesn't make sense).

Reply:
i added an update LastGossipedHeight inside UpdateTargetHeight, this will cause full-nodes also update LastGossipedHeight when retrieving form p2p or DA. this way, when rotating the value will be updated